parser/src/test/scala/hmda/parser/fi/lar/LarGenerators.scala

```diff
@@ -91,8 +91,7 @@ trait LarGenerators extends FIGenerators {
       "30", "31", "32", "33", "34", "35", "36", "37", "38", "39",
       "40", "41", "42", "44", "45", "46", "47", "48", "49",
       "50", "51", "53", "54", "55", "56",
-      "60", "66", "69",
-      "72", "74", "78"
```

Comment:
@lesgou I see. So the CBSA contains every state code that can file HMDA? And it doesn't have any extra locations that shouldn't file HMDA?

@jmarin Would you update the comment on line 86 of this file to reflect where the list actually comes from?